app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,19 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
+
+    if user
+      login user
+      redirect_to announcements_path
+    else
+      flash[:danger] = "Please enter a valid email address!"
+      redirect_to login_path
+    end
+  end
+
+  def destroy
```

Comment:
just curious, there is no way to log out?

Reply:
Not in this pr, but it will soon be added once we write those tests